### PR TITLE
Momchil/merge develop in 26

### PR DESCRIPTION
**Pull request by momchil-flex**

### CODE REVIEW

**Comment by tylerflex** [comment]
Hm, so we're going to basically apply black via this PR? if so should we also update the black version in pyproject.toml?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1063,7 +1074,8 @@ which fields are to be projected is now determined automatically based on the me
 - Job and Batch classes for better simulation handling (eventually to fully replace webapi functions).
 - A large number of small improvements and bug fixes.
 
-[Unreleased]: https://github.com/flexcompute/tidy3d/compare/v2.5.0...develop
+[Unreleased]: https://github.com/flexcompute/tidy3d/compare/v2.5.1...develop
```
Comment (tylerflex) [inline CHANGELOG.md:1077]: hm, probably this should be `v2.5.1...pre/2.6` (although I dont recall normally changing this might be good to)

**Review by tylerflex — APPROVED** [review]
looks good to me, one super minor comment about changelog link

File: CHANGELOG.md (hunk 1 above)
Comment (momchil-flex) [reply CHANGELOG.md:1077]: This just comes from develop now. I think it's fine yeah we don't normally put pre/ link there.

**Review by tylerflex — APPROVED** [review]
k should hopefully be all set now

File: CHANGELOG.md (hunk 1 above)
Comment (momchil-flex) [reply CHANGELOG.md:1077]: Actually I will make this update but only when finalizing the CHANGELOG for 2.6.0rc1, not now.

Comment (tylerflex) [reply CHANGELOG.md:1077]: yea I tried out the Link and it seemed to work fine. we can do it then.

**Pull request merged**